rules/five-user-experiences-of-reporting-services/rule.md

```diff
@@ -0,0 +1,44 @@
+---
+type: rule
+archivedreason: Only showing example screenshots
+title: Do you know the 5 user experiences of Reporting Services?
+guid: b66277bc-0091-402c-8d9b-ee3907eae909
+uri: five-user-experiences-of-reporting-services
```

Comment:
1. Please add this rule to a category 🙏 (if it isn't already)

Reply:
Done - added to rules to better reporting services